index.bs

```diff
@@ -328,12 +328,8 @@ Rights of Members</h4>
 	</ul>
 
 	The rights and benefits of W3C membership are contingent upon conformance to the processes described in this document.
-	The vast majority of W3C Members faithfully follow the spirit as well as the letter of these processes.
-	When serious and/or repeated violations do occur,
-	and repeated attempts to address these violations have not resolved the situation,
-	the [=Director=] <em class="rfc2119">may</em> take disciplinary action.
-	Arbitration in the case of further disagreement is governed by paragraph 19 of the Membership Agreement [[MEMBER-AGREEMENT]].
-	Refer to the <a href="https://www.w3.org/2002/09/discipline">Guidelines for Disciplinary Action</a> [[DISCIPLINARY-GL]].
+	Arbitration is governed by paragraph 19 of the Membership Agreement [[MEMBER-AGREEMENT]].
```

Comment:
delete: "Arbitration is governed by paragraph 19 of the Membership Agreement [[MEMBER-AGREEMENT]]." as it doesn't add anything here.